File: Level-0/84.java
class Solution {
    public int solution(String my_string) {
        int answer = 0;
        String[] s = my_string.split(" ");
        boolean bool = false;
        for(int i = 0; i < s.length; i++){
            if(s[i].equals("+")){
                bool = false;
            }else if(s[i].equals("-")){
                bool = true;
            }else{
                int n = Integer.parseInt(s[i]);
                if(bool){
                    answer -= n;
                }else{
                    answer += n;
                }
            }
        }
        return answer;
    }
}

File: Level-0/84.js
function solution(my_string) {
    let answer = 0;
    let oper = '';
    my_string.split(' ').forEach((v, i) => {
        if(i === 0) {
            answer += parseInt(v);
        }else {
            if(v === '+' || v === '-') {
                oper = v;
            }else {
                if(oper === '+') {
                    answer += parseInt(v);
                }else if(oper === '-') {
                    answer -= parseInt(v);
                }
            }
        }
    });
    return answer;
}
